### PR TITLE
Documentation for StackAdapt Segment Server-to-Server Destination

### PR DESCRIPTION
**Pull request by AlliterativeAlice**
# This PR will not be merged, once approved internally by StackAdapt stakeholders a new PR will be made and submitted to Segment

## Why?

[IDE-1649](https://stackadapt.atlassian.net/browse/IDE-1649)

As per Segment’s [contribution guide](https://github.com/segmentio/action-destinations/blob/main/CONTRIBUTING.md#write-documentation), we need to write documentation on how to use our destination before it can be published to customers.

## What?

Documentation on how customers can set up the StackAdapt Destination in Segment, including examples of how to create conversion events that will fire on forwarded Segment events.

## Testing

The documentation repo can be checked out an run locally by following the [developer guide](https://github.com/segmentio/segment-docs/blob/develop/devguide.md). To help save reviewers this trouble I've attached a screenshot of the documentation page as it appears when the docs site is run locally.

[IDE-1649]: https://stackadapt.atlassian.net/browse/IDE-1649?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

localhost_4000_docs_connections_destinations_catalog_actions-stackadapt_ (1)

### CODE REVIEW

File: src/connections/destinations/catalog/actions-stackadapt/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    }
+  ]
+});
+```
+
+### URL Rules
+
+If you are using URL rules, these will be matched whenever Segment sends an event to StackAdapt with a `url` matching the URL rule. This should be accomplished by the page event Segment automatically fires when a page is viewed, so setup of URL rules should be identical to setting up URL rules with the StackAdapt pixel.
+
+{% include components/actions-fields.html %}
```
Comment (AlliterativeAlice) [inline src/connections/destinations/catalog/actions-stackadapt/index.md:112]: Segment requests we include this component in their [documentation template](https://raw.githubusercontent.com/segmentio/action-destinations/main/docs/doc-template-new.md). It doesn't render properly currently but as I understand it, it should once our destination is public.

**Review by jelena-n — APPROVED** [review]
Looks good to me :-)

**Comment by Arun-Issac** [comment]
Do you think there's a need to obfuscate the universal pixel ID shown in the screenshot? Is that from a real account on the platform? Just wondering if someone may utilize it to send garbage data to us

**Comment by AlliterativeAlice** [comment]
> Do you think there's a need to obfuscate the universal pixel ID shown in the screenshot? Is that from a real account on the platform? Just wondering if someone may utilize it to send garbage data to us

I modified the pixel ID after taking the screenshot so that it's not a real universal pixel ID.

**Review by Arun-Issac — APPROVED** [review]
looks good

**Review by omfgitsjack — APPROVED** [review]
lgtm!

**Pull request closed without merging**